code/game/objects/structures/crates_lockers/closets/utility_closets.dm

```diff
@@ -139,10 +139,7 @@
 	new /obj/item/geiger_counter(src)
 	new /obj/item/clothing/suit/radiation(src)
 	new /obj/item/clothing/head/radiation(src)
-	if(prob(50))
-		new /obj/item/storage/firstaid/radbgone(src)
-	else
-		new /obj/item/storage/pill_bottle/antirad(src)
+	new /obj/item/storage/pill_bottle/antirad(src)
```

Comment:
We may need to consider another means of getting radbgone medkits if there aren't any.
Maybe cargo ordered.

Reply:
I'll add a medical crate for it